flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -2332,14 +2332,24 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
 
     public static final BuiltInFunctionDefinition TO_TIMESTAMP_LTZ =
             BuiltInFunctionDefinition.newBuilder()
-                    .name("toTimestampLtz")
-                    .sqlName("TO_TIMESTAMP_LTZ")
+                    .name("TO_TIMESTAMP_LTZ")
```

Comment:
looks like we are changing the name of an existing function - I assume this will impact migration. Don't we need to deprecate the old function and add the new one with the new name? And then put thjis in the documentation

Reply:
Hi @davidradl , it shouldn't impact the migration, because we are rewriting scala code into java, and the existing tests passed just fine.

Reply:
Hi @snuyanzin , gave it a try and spent much time on it. Still, for some reason, if I revert the two names back, when I run my new java tests, I got errors like 
`SQL validation failed. From line 1, column 8 to line 1, column 81: Cannot apply 'TO_TIMESTAMP_LTZ' to arguments of type 'TO_TIMESTAMP_LTZ(<CHAR(25)>, <CHAR(25)>)'. Supported form(s): 'TO_TIMESTAMP_LTZ(<NUMERIC>, <INTEGER>)'`

Could you shed some light on this? It seems like the SQL validation doesn't pick up my change.

Reply:
Reverted the change and made all existing tests pass.

Reply:
After some research, the new Java stack only has one "name", eg. [here](https://github.com/apache/flink/blob/01a2b28160f70a12d7200790acde53a9e910339a/flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java#L237). 
We are not changing the api name, it recognizes the api name [here](https://github.com/apache/flink/pull/25763/files#diff-d2ce68433255086dfdc898b9fd6636186044115a1280d39d6d04ebfd99933e94R377-R422).